sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -892,3 +898,186 @@ def test_fit_predict(self, estimator_instance, n_columns):
         cutoff = get_cutoff(y_train, return_index=True)
         _assert_correct_pred_time_index(y_pred.index, cutoff, fh)
         _assert_correct_columns(y_pred, y_train)
+
+
+class TestAllGlobalForecasters(TestAllObjects):
```

Comment:
Please correct me if I am wrong. If I have seen it correctly, you passed a copy of y_train to the predict methods. I am wondering if we can make this more general:
* E.g. renaming the columns of y_test so that they are different from y_train and check if the current column names are returned.
* Provide a different y_test than the y_train to the test and check if the results of both are different

Reply:
@fkiraly maybe you have a comment on that

Reply:
Renaming the column is added to `test_global_forecasting_series`

Train test spliting with overlap is added to test predicting on instances both included and not included in `fit`.